.github/actions/spelling/patterns.txt

```diff
@@ -1,19 +1,50 @@
 # See https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns
 
+# Automatically suggested patterns
```

Comment:
This set is based on the refreshed items above. I could move them down below so that the diff is clearer...